tests/wpt/tests/content-security-policy/reporting/report-multiple-violations-02.html

```diff
@@ -12,7 +12,7 @@
 <body>
   <script>
     for (var i = 0; i<5; i++)
-      setTimeout("document.body.innerHTML += ('<p>PASS: setTimeout #" + i + " executed.');", 0);
+      setTimeout("document.body.innerHTML += ('<div>PASS: setTimeout #" + i + " executed.</div>');", 0);
```

Comment:
This change confuses me, since other browsers don't time out: https://wpt.fyi/results/content-security-policy/reporting/report-multiple-violations-02.html?label=master&label=experimental&aligned&q=content-security-policy%2Freporting%2Freport-multiple-violations-02.html

Do we know why this makes a difference in Servo?

Reply:
Hm, when I was debugging this test it was failing with a testrunner error. Now I can't reproduce that anymore. I will revert these changes.